### ISSUE TITLE
Example for multiple file upload on submit button press

### ISSUE THREAD

**Issue by jculverwell** [blueimp/jQuery-File-Upload#1867]
First of all thanks for this superb component, it's by far the best upload component I've used.

I am trying to figure out how to submit two files at once when a submit button is pressed and unfortunately when I press the submit button I am only getting one file coming through.  I've read the API documentation and also various messages in this forum but I haven't found and end to end solution and I'm sure my basic knowledge of javascript is holding me back. 

You can find my code here http://jsbin.com/welcome/50297/edit which is where I have got to.

Basically you can see I am overriding the add function and hooking up a click handler to the submit button.  I haven't included my server code but I've verified if that if I post the form not using ajax that the two files comes through, as mentioned my issue is if I post over ajax then only one file comes through.

I would appreciate any pointers, I'm sure other people will find this useful as well.  

If possible I would like the solution to work in IE8+ and just use the the basic plugin.

Thanks for any help.
